corehq/apps/export/forms.py

```diff
@@ -1029,6 +1041,7 @@ def get_case_filter(self, mobile_user_and_group_slugs, can_access_all_locations,
             accessible_location_ids,
             self.dynamic_filter_class.show_all_data(mobile_user_and_group_slugs),
             self.dynamic_filter_class.show_project_data(mobile_user_and_group_slugs),
+            self.dynamic_filter_class.show_deactivated_data(mobile_user_and_group_slugs),
```

Comment:
I added this because of the modification I made to the inputs [here](https://github.com/dimagi/commcare-hq/pull/22179/commits/8ad32ae1fa0fffb8bccb18a2c909ce79f4968c4c#diff-628ce8b77792cabb27a638d59a0b064fR765). I tested doing an export with the deactivated filter, I think that tests this code but I'd like a second opinion on that!

Reply:
Don't have enough context here to plus one. If there is an integration test somewhere for this that might be better way instead.